distribution/docs/src/main/resources/_contents/_core-concepts/registry-intro-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+The Registry Application serves as an index of registry nodes and their information, including service bindings, configurations and supplemental details.
+
+Each registry has the capability to serve as an index of information about a network of registries which, in turn, can be used to connect across a network of ${branding}s and other data sources.
+Registries communicate with each other through the CSW endpoint and each registry node is converted into a registry metacard to be stored in the catalog.
+When a registry is subscribed to or published from, it sends the details of one or more nodes to another registry.
+
+=== Identity Node
+
+The ${ddf-registry} is initially comprised of a single registry node, refered to as the *identity*, which represents the registry's primary configuration.
+
+=== Subscription
```

Comment:
We need to work on the registry terminology at some point.  We uses nodes here but do not refer to other systems we federate to as nodes any where else.  I wonder if there will be confusion about the difference between subscription of registries and events.